department-of-veterans-affairs/vets-website
# Remove Express Care and past appointments flags

## Description
This PR:

- Removes the Express Care flag
- Updates the Express Care CTA to be gated by the express care new request flag 
- Removes the past appointments flag
- Removes any code made unnecessary by removing the flags
- Fixes a couple of flaky unit tests I ran into
- Removes an unused component, FutureAppointmentsListV2.jsx (new one is UpcomingAppointmentsList)

How to test:

- Disable the va_online_scheduling_express_care and va_online_scheduling_past flags
- Open the app and verify that the Express call to action still displays, as well as the past and express care tabs

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Express care and past appointments still work

## Definition of done
